reliability-engineering/pipelines/tasks/generate-deploy-info-pipelines-pipeline.yml

```diff
@@ -0,0 +1,59 @@
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: concourse/concourse-pipeline-resource
+    tag: "6.0.0"
+inputs:
+- name: tech-ops-private
+outputs:
+- name: deploy-info-pipelines-pipeline
+params:
+  DEPLOYMENT_NAME:
+  CONCOURSE_USERNAME:
+  CONCOURSE_PASSWORD:
+run:
+  path: /bin/bash
+  args:
+  - -euo
+  - pipefail
+  - -c
+  - |
+    mkdir -p deploy-info-pipelines-pipeline
+    export PATH="$PATH:/opt/resource"
+    fly -t concourse login -c "https://${DEPLOYMENT_NAME}.gds-reliability.engineering" -u $CONCOURSE_USERNAME -p $CONCOURSE_PASSWORD -n $CONCOURSE_USERNAME
+    fly -t concourse sync
+    fly -t concourse teams --json | jq '{
```

Comment:
why is this `jq` rather than `yq`? It feels weird given that a) we use `yq` in the same task, and b) concourse pipelines are normally viewed as YAML.

That said, this is weird but not a blocker.

Reply:
Because technically what fly is outputting is actually JSON. Since JSON is a subset of YAML we could just use `yq`, it doesn't matter a great deal. I think yq would be using jq under the hood anyway?